### ISSUE TITLE
[css-text-4] change "Creates 1/4em extra spacing between" to "Creates extra spacing between"

### ISSUE THREAD

**Issue by MurakamiShinyu** [w3c/csswg-drafts#7050]
https://drafts.csswg.org/css-text-4/#text-spacing-property

This proposal was found in the following issue:

- https://github.com/w3c/csswg-drafts/issues/3229

> 1. In the above description of `ideographic-alpha` and `ideographic-numeric`, change "Creates 1/4em extra spacing between" to "Creates extra spacing between."

and this issue was resolved as:

> RESOLVED: Add a new text-spacing:auto value, which is not the initial value.

I am ok with adding the `auto` value, but I think it's not sufficient. Changing "Creates 1/4em extra spacing between" to "Creates extra spacing between" is still necessary.

The "1/4em extra spacing" is not the best choice in many cases, and narrower space such as 1/6em is preferred. Apple iOS seems to use 1/6em for ideographic-alpha/numeric spacing.

For the `auto` value, UA can choose such better spacing value for ideographic-alpha/numeric spacing, but stylesheet authors cannot use the `auto` value when they need to choose specific text-spacing style, e.g., preference between `allow-end` and `trim-end`, or `space-first` and `trim-start`. So the spacing value of `ideographic-alpha` and `ideographic-numeric` should be same as the spacing value for ideographic-alpha/numeric spacing with `auto`.

---

FYI, [Vivliostyle](https://vivliostyle.org/) supports the `text-spacing` property, with `auto` value which is equivalent to `trim-start trim-end trim-adjacent ideograph-alpha ideograph-numeric` and the `ideograph-alpha` and `ideograph-numeric` create 1/6em space. See https://docs.vivliostyle.org/#/supported-css-features#css-text-4

**Comment by xfq** [comment]
Indeed, different software (Adobe InDesign, Microsoft Word, Core Text, etc.) and different authors/publishers may have different preferences, and the point is that there is extra spacing between ideographs and non-ideographic letters/numerals. I feel like we can move "1/4em" to a non-normative note as a guideline to implementers who don't know how much space to add, instead of making it a hard and fast normative rule.

See also: https://github.com/w3c/jlreq/issues/163

**Comment by xfq** [comment]
And since IIRC no browsers have implemented this property yet (except for IE, but IIRC the property names are different), I don't think we'll have any web compat issues.

**Comment by fantasai** [comment]
Accepting this under editor's discretion. @MurakamiShinyu @xfq Let me know if the edits look appropriate! Thanks!

**Comment by MurakamiShinyu** [comment]
@fantasai Yes, looks good to me. Thanks!

**Comment by xfq** [comment]
It looks good to me. Thank you, @fantasai!

**Comment by 070889533** [comment]
> https://drafts.csswg.org/css-text-4/#text-spacing-property
> 
> This proposal was found in the following issue:
> 
> * [[css-text-4] Text-spacing is too strict #3229](https://github.com/w3c/csswg-drafts/issues/3229)
> 
> > 1. In the above description of `ideographic-alpha` and `ideographic-numeric`, change "Creates 1/4em extra spacing between" to "Creates extra spacing between."
> 
> and this issue was resolved as:
> 
> > RESOLVED: Add a new text-spacing:auto value, which is not the initial value.
> 
> I am ok with adding the `auto` value, but I think it's not sufficient. Changing "Creates 1/4em extra spacing between" to "Creates extra spacing between" is still necessary.
> 
> The "1/4em extra spacing" is not the best choice in many cases, and narrower space such as 1/6em is preferred. Apple iOS seems to use 1/6em for ideographic-alpha/numeric spacing.
> 
> For the `auto` value, UA can choose such better spacing value for ideographic-alpha/numeric spacing, but stylesheet authors cannot use the `auto` value when they need to choose specific text-spacing style, e.g., preference between `allow-end` and `trim-end`, or `space-first` and `trim-start`. So the spacing value of `ideographic-alpha` and `ideographic-numeric` should be same as the spacing value for ideographic-alpha/numeric spacing with `auto`.
> 
> FYI, [Vivliostyle](https://vivliostyle.org/) supports the `text-spacing` property, with `auto` value which is equivalent to `trim-start trim-end trim-adjacent ideograph-alpha ideograph-numeric` and the `ideograph-alpha` and `ideograph-numeric` create 1/6em space. See https://docs.vivliostyle.org/#/supported-css-features#css-text-4

https://github.com/w3c/csswg-drafts/issues/7050#issue-1138440614
